### DISCUSSION TITLE
Silverlit protocol?

### DISCUSSION THREAD

**Discussion by Eagleowl1** [pascallanger/DIY-Multiprotocol-TX-Module#737] [General]
Hello,
I recently acquired a module from iRange: iRX4 lite 
I am using it together with a Frsky X20 on Ethos.
From RCgroup I downloaded an impressive long list of protocols defined a LUA script.
On my side everything is working fine. (DSMX and FHSS)

I have not found a protocol for Silverlit range of 2.4GHz toys.

My question: does such a protocol exist? or can another compatible protocol be used to bind with the Silverlit toy?

Thanks

**Comment by shafr** [comment]
@Eagleowl1 can you please share link to that LUA script ?

**Reply by Eagleowl1** [reply]
This is the one I am using successfully with Frsky X20S
See #20627

https://www.rcgroups.com/forums/showthread.php?3703807-FrSky-X20-X20S-X20HD-Tandem-24-Channels-with-ETHOS/page1376#post49982683

Silverlit is not on the list.
I was wondering if Silverlit was using somebody else protocol?

**Comment by shafr** [comment]
As always - you need to either disassemble and look what chip is inside or send receiver & transmitter to Pascal for research (or donate so he could buy his own set).
Apparently there is only few chip manufacturers, but for some reason every company (or even every model) has their own protocol which needs to be implemented in multi-protocol.
